environment.boot

```diff
@@ -1,4 +1,4 @@
 azul_python_version=3.11.9
 azul_python_image=docker.io/library/python:3.11.9-bullseye
-azul_docker_version=25.0.5
-azul_terraform_version=1.6.6
+azul_docker_version=27.0.3
```

Comment:
We should probably have instructed developers to ensure they have this version installed locally.